packages/api/src/aggregate/WhereClause.ts

```diff
@@ -168,7 +170,21 @@ type FilterFor<PD extends ObjectMetadata.Property> = PD["multiplicity"] extends
     : PD["type"] extends "geopoint" | "geoshape" ? GeoFilter
     : PD["type"] extends "datetime" | "timestamp" ? DatetimeFilter
     : PD["type"] extends "boolean" ? BooleanFilter
-    : NumberFilter); // FIXME we need to represent all types
+    : PD["type"] extends
```

Comment:
Fixed this so we don't default to number filter for properties like geotimeseries or timeseries